### PR TITLE
Kevin/site perf2

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

Opening against master instead of stable 
https://github.com/gitcoinco/web/pull/4800

### CODE REVIEW

File: app/assets/v2/css/home.css [hunk 1]
```diff
@@ -223,7 +223,7 @@ img.play {
   cursor: pointer;
   --padding: 40px;
   background-color: var(--gc-green-hover);
-  background-image: url('../images/home/bg-video-preview.svg');
+
```
Comment (danlipert) [inline app/assets/v2/css/home.css]: do we want a blank line here?

**Comment by thelostone-mc** [comment]
@danlipert 

> Trying to review this is crashing my computer - can we split this into assets vs code changes perhaps?

How do I do that? removing the html changes manually ?  :P

File: app/assets/v2/css/home.css (hunk 1 above)
Comment (thelostone-mc) [reply app/assets/v2/css/home.css]: fixed 🙌

**Comment by octavioamu** [comment]
image
image

**Comment by thelostone-mc** [comment]
Yeah I had those same issues !
@owocki I'm not sure what else could be broken.
There isn't a way to be sure unless each and every image is verified ! 

We could merge and fix stuff as we find them / we can drop this PR.
I'm leaning towards the former cause of the benefits we'd get

cc @danlipert / @octavioamu

**Comment by thelostone-mc** [comment]
@owocki we got it in without splitting it up! chill we took care of it

**Comment by owocki** [comment]
@thelostone-mc cool; i answered all the questions on https://github.com/gitcoinco/web/pull/4826 -- hoping to get dem code changes in too!

**Pull request merged**